rockstarVR/rockstar.html

```diff
@@ -0,0 +1,36 @@
+---
+title: RockstarVR | SeeSaw Labs
+layout: default
+permalink: /rockstarVR/
```

Comment:
@mjbarnes88 and @luigicr18 

I think the name for this url should be more SEO friendly.
Maybe /rockstar-virtual-reality/ or just /virtual-reality/ since the rockstart doesn't say much about our work.

@luigicr18 please add the new page in the sitemap.xml